client/jetpack-connect/utils.js

```diff
@@ -126,3 +126,24 @@ export function parseAuthorizationQuery( query ) {
 	}
 	return null;
 }
+
+export function getPartnerSlugFromId( partnerId ) {
+	switch ( parseInt( partnerId, 10 ) ) {
+		case 51945:
+		case 51946:
+			return 'dreamhost';
+		case 49615:
+		case 49640:
+			return 'pressable';
+		case 57152:
+		case 57733:
+			return 'milesweb';
+		case 41986:
+		case 42000:
+			return 'bluehost';
+		case 51652: // Clients used for testing.
+			return 'dreamhost';
```

Comment:
These slugs are now spread across several files. It might makes sense to start using shared constants here:

https://github.com/Automattic/wp-calypso/blob/6d893be24b006e0c49457938a82721659abe432f/client/jetpack-connect/constants.js#L2-L4

Reply:
I've thought about this a bit, and I'm not sure that it would be any more beneficial to use constants for the hosting partner slugs, primary I don't anticipate the slugs changing and also because the constant would end up having the partner slug in it anyways.
 
For example, the slug we use for `bluehost` would be something like: `PARTNER_SLUG_BLUEHOST` or `BLUEHOST_PARTNER_SLUG`. In both cases, `bluehost` is in the slug. So, at this point, we're adding characters and requiring a file for something that likely won't change.

I think the constants are a great idea for the three cases you link to above. `JETPACK_MINIMUM_WORDPRESS_VERSION`, `JPC_PATH_PLANS`, and `JPC_PATH_REMOTE_INSTALL` are all subject to changing, especially the minimum WordPress version. 😄

Reply:
👍 Sure, not a blocker, just a thought. I try to avoid matching literal strings as it's easy to mistype something and introduce bugs. This is working fine and I'm happy to leave it.